config/routes.rb

```diff
@@ -2,6 +2,7 @@
   resources :errors, only: [:show]
   resources :teams, only: [:index, :show, :create, :update]
   resources :team_users, only: [:create, :destroy, :update]
+  get "teams/typeahead/:id/:query" => "teams#typeahead", :defaults => { format: "json" }
```

Comment:
Wouldn't it be better to handle this as member action?
```
resources :teams, only: [:index, :show, :create, :update]
  member do
    get :typeahead
  end
end
```
@flavio @mssola : any opinion?